CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # splat Release Notes
 
+## 0.9.0: The Big Update
+### Introducing [spimdisasm](https://github.com/Decompollaborate/spimdisasm)!
+* Thanks to [AngheloAlf](https://github.com/AngheloAlf), we now have a much better MIPS disassembler in splat! spimdisasm has much better hi/lo matching, much lower ram usage, and is just overall grood.
```

Comment:
I figured `grood` was an ethan-ism from knowing your style a bit but it may be confusing to others :smile:

Reply:
oh lol thanks for mentioning that. I think I wrote that sentence as mostly a placeholder and intended to remove "grood" later on 😁